### DISCUSSION TITLE
Add translation for system collections

### DISCUSSION THREAD

**Discussion by nicolaschapeau** [directus/directus#15329] [Archive]
Hello,

I created a system that rely on field translation to render the correct name for each field on my frontend.
![image](https://user-images.githubusercontent.com/32775238/187537325-83d6b1ed-f8c4-48fc-92b2-17df43079c01.png)

But I couldn't find a way to translate system fields since their modification is locked.

Is there any way to translate in this way system fields ?
![image](https://user-images.githubusercontent.com/32775238/187537551-049b667a-7635-4b41-8526-a929f1e1bd88.png)

I think it's possible with the API, but it would be longer 😅

Regards, Nicolas.

**Comment by rijkvanzanten** [comment]
There's currently no way to add custom names for the system collections and their fields

**Comment by contraintuitiv** [comment]
Hi, I stumbled over the same problem. Is there a way now? It's very irritating for my editors that it's "Directus Files" in the Block-Editor, and not a german version.
